composer.json

```
@@ -20,32 +20,33 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
         "az-digital/arizona-bootstrap": "2.0.5",
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.7",
+        "drupal/auto_entitylabel": "3.0-beta3",
         "drupal/background_image_formatter": "1.7",
         "drupal/bootstrap_barrio": "4.30",
         "drupal/cas": "1.7",
+        "drupal/ckeditor_bs_grid": "2.0.3",
+        "drupal/coffee": "1.0",
         "drupal/core-recommended": "9.0.9",
         "drupal/ctools": "3.4",
+        "drupal/draggableviews": "2.0.0",
         "drupal/easy_breadcrumb": "1.13",
+        "drupal/embed": "1.4",
+        "drupal/entity_embed": "1.1",
         "drupal/externalauth": "1.3",
         "drupal/field_group": "3.1",
+        "drupal/media_library_form_element": "2.0",
         "drupal/media_library_theme_reset": "1.0",
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
+        "drupal/paragraphs": "1.12",
         "drupal/pathauto": "1.8",
         "drupal/token": "1.7",
-        "drupal/paragraphs": "1.12",
-        "drupal/entity_embed": "1.1",
-        "drupal/embed": "1.4",
-        "drupal/viewsreference": "2.0-beta2",
-        "drupal/media_library_form_element": "2.0",
-        "drupal/ckeditor_bs_grid": "2.0.3",
-        "drupal/auto_entitylabel": "3.0-beta3",
         "drupal/views_bootstrap": "4.3",
-        "drupal/draggableviews": "2.0.0"
+        "drupal/viewsreference": "2.0-beta2",
+        "php": ">=7.0"
```

Per comment above, I think we actually want to leave this php line at the top of the list.

```suggestion
        "drupal/viewsreference": "2.0-beta2"
```